Source/KSCrash/Recording/KSSystemCapabilities.h

```diff
@@ -38,7 +38,8 @@
 #define KSCRASH_HOST_IOS (KSCRASH_HOST_APPLE && TARGET_OS_IOS)
 #define KSCRASH_HOST_TV (KSCRASH_HOST_APPLE && TARGET_OS_TV)
 #define KSCRASH_HOST_WATCH (KSCRASH_HOST_APPLE && TARGET_OS_WATCH)
-#define KSCRASH_HOST_MAC (KSCRASH_HOST_APPLE && TARGET_OS_MAC && !(TARGET_OS_IOS || TARGET_OS_TV || TARGET_OS_WATCH))
+#define KSCRASH_HOST_VISION (KSCRASH_HOST_APPLE && TARGET_OS_VISION)
+#define KSCRASH_HOST_MAC (KSCRASH_HOST_APPLE && TARGET_OS_MAC && !(TARGET_OS_IOS || TARGET_OS_TV || TARGET_OS_WATCH || TARGET_OS_VISION))
```

Comment:
What about `TARGET_OS_MACCATALYST`?

Reply:
Good question, but not related to this PR. Feel free to investigate